### ISSUE TITLE
More information about “replacing the opcodes” bit

### ISSUE THREAD

**Issue by miohtama** [ethereum-optimism/optimism-tutorial#5]
In README we have a bit

``` 
Ethereum are required to replace certain EVM opcodes with custom behavior.
```

What would be the source material for this to know what actually happens here? Would be nice to link from README, so the reader does not need to guess.

**Comment by kfichter** [comment]
Closed by #6

**Comment by kfichter** [comment]
Thank you for the help here @miohtama!
